### PR TITLE
Consolidate and iterate performance reporting

### PR DESCRIPTION
**Pull request by benilovj**
## Context

The current performance dashboard makes it hard to see what's happening post-submission (how many successful and unsuccessful candidates there are) at a glance.

## Changes proposed in this pull request

Iterate the service performance dashboard:

- consolidate candidate and application form reporting into just candidate reporting, because it's effectively the same thing until Apply 2 is added
- add more focus to how many applications are in pre- and post-submission.
- make it clearer how many applications are in important end-states (offers accepted or unsuccessful)

Before:

image

After:

image

## Guidance to review

- is the visual design (layout and colours) OK?
- for the top-level metrics, is it clear what the number is from the labels?

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by duncanjbrown** [comment]
Great! I like very much seeing each layer as the sum of the layer below, which holds for the first two rows (183 = 54 +129). This isn't the case in the bottom-right.

Should we drop the "Dashboard" from the title? It seems a bit redundant

**Comment by duncanjbrown** [comment]
What @tijmenb said 😂

**Comment by fofr** [comment]
Maybe not orange for submitted. 🤔

**Comment by benilovj** [comment]
Thanks all for the feedback – see the updated screenshot:

image

Changes:

- I've pared down the colours, they can be iterated by and actual designer on a later PR
- I've updated the 3rd row of metrics so it adds up to the 2nd row, and the fonts are smaller
- I've removed the word 'dashboard' from the title

File: app/views/support_interface/performance/index.html.erb [hunk 1]
```diff
@@ -1,50 +1,78 @@
-<% content_for :title, 'Service performance dashboard' %>
+<% content_for :title, 'Service performance' %>
```
Comment (fofr) [inline app/views/support_interface/performance/index.html.erb]: :+1:

**Pull request merged**